scripts/linters/pre_commit_linter.py

```diff
@@ -422,6 +422,23 @@ def _print_complete_summary_of_errors(all_messages):
             python_utils.PRINT(message)
 
 
+def _get_task_output(task, semaphore):
```

Comment:
Looking back at PR description, you would need to pass in `all_messages` in here too, since the control of the semaphore is intended to work on `all_messages`. This function would then have to append `task.output` to it. You wouldn't need to return anything since the function would be modifying `all_messages`.

Reply:
Done.